src/org/labkey/test/tests/ExpTest.java

```diff
@@ -163,17 +166,17 @@ public void testSteps()
         waitForElement(Locator.xpath("//span[contains(text(), 'Reset to Default')]"), defaultWaitForPage);
         int lastFieldIndex = getElementCount(Locator.xpath("//input[starts-with(@name, 'ff_label')]")) - 1;
         Locator lastField = Locator.xpath("//input[@name='ff_label" + lastFieldIndex + "']");
-        click(lastField);
-        click(Locator.xpath("//span").append(Locator.lkButton("Alias Field")));
-        selectOptionByText(Locator.name("sourceColumn"), "RowId");
-        click(Locator.xpath("//span").append(Locator.lkButton("OK")));
+        domainRow = designerPage.fieldsPanel().getField(lastFieldIndex);
+        clickButton("Alias Field");
+        SelectWrapper.Select(Locator.tagWithAttribute("select", "name", "aliasField")).findWhenNeeded(getDriver()).selectByVisibleText("RowId");
+        clickButton("OK");
 
         // Make it a lookup into our custom query
         editor = PropertiesEditor.PropertiesEditor(getDriver()).withTitleContaining("Metadata Properties").find();
         int fieldCount = getElementCount(Locator.xpath("//input[contains(@name, 'ff_type')]"));
```

Comment:
Same comment as above related to the "ff_type" input name. I don't think that will exist for the new designer UI. Also, shouldn't the use of the PropertiesEditor be removed completely?